bench/hist/Main.hs

```diff
@@ -290,11 +290,11 @@ buildGhcide Stack out =
         <> " build ghcide:ghcide --copy-bins --ghc-options -rtsopts"
 
 
-findGhc :: BuildSystem -> Action FilePath
-findGhc Cabal =
+findGhc :: FilePath -> BuildSystem -> Action FilePath
+findGhc _cwd Cabal =
     liftIO $ fromMaybe (error "ghc is not in the PATH") <$> findExecutable "ghc"
-findGhc Stack = do
-    Stdout ghcLoc <- cmd (s "stack exec which ghc")
+findGhc cwd Stack = do
+    Stdout ghcLoc <- cmd [Cwd cwd] (s "stack exec which ghc")
```

Comment:
Should we undo this change? Or is it worth keeping even though it didn't work out?

Reply:
We should keep it. it's a correctness change, as the `ghc` binary used by stack is context dependent